mapper/census_test.go

```diff
@@ -593,9 +608,19 @@ func TestCreateCensusDatasetLandingPage(t *testing.T) {
 					fDims,
 					serviceMessage,
 					emergencyBanner)
-				So(page.DatasetLandingPage.HasDownloads, ShouldBeTrue)
+
+				Convey("HasDownloads set to true", func() {
+					So(page.DatasetLandingPage.HasDownloads, ShouldBeTrue)
+				})
+
+				Convey("Downloads are sorted by fixed extension order", func() {
+					So(page.Version.Downloads[0].Extension, ShouldEqual, "xls")
+					So(page.Version.Downloads[1].Extension, ShouldEqual, "csv")
+					So(page.Version.Downloads[2].Extension, ShouldEqual, "txt")
+					So(page.Version.Downloads[3].Extension, ShouldEqual, "csvw")
+				})
```

Comment:
Would be good to have a test to prove the ordering works when there's only three downloads as `xls` won't always be available

Reply:
added for dataset and filter outputs